_includes/components/sidebar.html

```diff
@@ -26,7 +26,7 @@
     {{ nav_footer_custom }}
   {% else %}
     <footer class="site-footer">
-      Content Produced by <a href="https://github.com/VerzatileDev">VerzatileDev</a> & <a href="https://github.com/JDSherbert">JDSherbert</a>
+      An Open source project hosted at: <a href="https://github.com/VerzatileDevOrg/Programming_HandBook">Programming HandBook</a>
```

Comment:
```suggestion
      An Open source project hosted at: <a href="https://github.com/VerzatileDevOrg/Programming_HandBook">Programming Handbook</a>
```
Really minor, but:
"Book" in HandBook" does not need to be capitalised. It's all one work!

Reply:
Agreed, thank you for pointing that out!